src/LocalCache_TEST.cc

```diff
@@ -242,13 +242,16 @@ TEST_F(LocalCacheTest, AllModels)
   std::set<std::string> uniqueNames;
   while (iter)
   {
+    std::cerr << iter->Identification().UniqueName() << std::endl;
     uniqueNames.insert(iter->Identification().UniqueName());
     ++iter;
   }
   EXPECT_EQ(9u, uniqueNames.size());
 
-  EXPECT_NE(uniqueNames.end(), uniqueNames.find(
-      "http://localhost:8001/alice/models/am1"));
+  auto toFind = ignition::common::joinPaths(
```

Comment:
uhmm, I don't think this is a good idea, on windows you will use `backslash` to join the URI strings

Reply:
I actually came up from looking at the `AllWorlds` test below, which had mostly the same logic, but was passing: https://github.com/ignitionrobotics/ign-fuel-tools/blob/3c901105bb90bfdca918ea0d89715ab6fd11bb88/src/LocalCache_TEST.cc#L385-L391

I don't know that it's the 100% correct solution, but I want to get CI green so I can do some heavier cleaning.